aimmo-game/simulation/avatar/avatar_manager.py

```diff
@@ -23,8 +23,39 @@ def remove_avatar(self, user_id):
 
     @property
     def avatars(self):
-        return self.avatars_by_id.viewvalues()
+        return list(self.avatars_by_id.viewvalues())
 
     @property
     def active_avatars(self):
         return [player for player in self.avatars]
+
+    def players_update(self):
+        """
+        To be called on a required update of players in the front-end of the game. This is a wrapper for player_list.
+        :return: A dictionary that holds the list generated by player_list.
+        """
+
+        def _player_list():
+            """
+            Used in the serialising process to provide updates of avatars to the front end.
+            :return A list with each element containing a dictionary of each player data.
+            """
+            list_of_players = []
```

Comment:
weird name

Reply:
What do you suggest? can't think of a better one

Reply:
just players should be enough.